JamitFoundationExample/JamitFoundationExample.xcodeproj/project.pbxproj

```diff
@@ -1115,65 +1146,70 @@
 /* End XCConfigurationList section */
 
 /* Begin XCRemoteSwiftPackageReference section */
-		1EBCCBC425F4F0BC00079E3F /* XCRemoteSwiftPackageReference "JamitFoundation" */ = {
+		C54191312860AD9C00E29C3B /* XCRemoteSwiftPackageReference "JamitFoundation" */ = {
 			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "https://github.com/JamitLabs/JamitFoundation";
```

Comment:
With the change to SSH, I got a build error for some reason that I needed to first trust the SSH server to fetch the dependency.

Is there any advantage for using SSH? If not, I'd suggest to stay with the HTTPS variant.